src/utils/window-console.js

```diff
@@ -6,15 +6,11 @@ import { stripIndent } from 'common-tags';
 import type { GetState } from '../types/store';
 import { getProfile, selectedThreadSelectors } from '../reducers/profile-view';
 
-// Provide a bit of typing hinting. Flow doesn't understand the dynamic nature of
-// Object.defineProperty and has no definitions around it.
-type DefineGetProperty<T> = (
-  object: Object,
-  key: string,
-  {
-    +get: () => T,
-  }
-) => void;
+// Despite providing a good libdef for Object.defineProperty, Flow still
+// special-cases the `value` property: if it's missing it throws an error. Using
+// this indirection seems to work around this issue.
+// See https://github.com/facebook/flow/issues/285
+const defineProperty = Object.defineProperty;
```

Comment:
Flow now has a nice definition in https://github.com/facebook/flow/blob/3b97b169be16fe63582a69721d6dc8661fb009ab/lib/core.js#L24-L31, but still throws an error because of some hard-coded special-case in its ML code. See the aforementioned issue to know more...